src/utils/api.ts

```diff
@@ -15,3 +15,19 @@ export async function postUser(user: User): Promise<User> {
   const result: User = await response.json();
   return result;
 }
+
+export async function verifyLogin(user: Partial<User>): Promise<User> {
```

Comment:
Not sure if I would call it `verifyLogin` because you actually try to login.
Maybe rename it to `login`?

Reply:
Yes, I agree and changed it to login. Thx!